src/pages/understand-stacks/running-testnet-node.md

```diff
@@ -23,6 +23,29 @@ This tutorial will walk you through the following steps:
 
 ## Requirements
 
+In order to run a node, some software and hardware requirements need to be considered.
+
+### Hardware
+
+Running a node has no specialized hardware requirements. People were successful at running a node on Raspberry Pis, for instance.
+Minimum requirements are moving targets due to the nature of the project and some factors should be considered:
+
+- compiling node sources locally requires processing and storage resources
+- replicating the chainstate requires significant memory allocation (1,3+ GB)
+- chainstate size grows over time, increasing the memory requirements
```

Comment:
I think there's some confusion here and in the prior point:

* as the chain grows, the *on-disk state* will grow over time. Memory requirements are relatively static (not a function of chain-length, mostly the mempool).
* "Replicating the chainstate requires significant memory allocation" --  I think what you may referring to here is memory requirements for *ingesting the Stacks 1.0 state* -- those are indeed higher (~2-GB RAM) but only relevant to nodes that are upgrading from Stacks 1.0 or participating in Stacks 2.0 launch. If you spin up a node post launch, the memory requirements to sync w/ the network would be similar to what it was in Krypton or Xenon, which is to say, not a lot.
- "1,3+ GB" -- typo, perhaps you meant "1-3 GB"?

Reply:
I'll remove the memory allocation point (2nd in the list) and alter the chain growth point. thanks for the feedback!